boards/nxp/ucans32k1sic/ucans32k1sic.dts

```diff
@@ -192,3 +192,19 @@
 	phys = <&can_phy1>;
 	status = "okay";
 };
+
+&adc0 {
+	interrupts = <39 0>;
+	clk-source = <0>;
+	clk-divider = <1>;
```

Comment:
move `interrupts`, `clk-source`, `clk-divider` to the soc dts, since it's not board configuration dependent